modules/nf-core/qsv/cat/main.nf

```diff
@@ -0,0 +1,45 @@
+process QSV_CAT {
+    tag "${meta.id}"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
+        ? 'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/86/864fcf1e6a250b588eaabd4df01334e44bc528fd29049c4fb24f7c56c8c33695/data'
+        : 'community.wave.seqera.io/library/qsv:5.1.0--9a6a0c23d3b279b5'}"
+
+    input:
+    tuple val(meta), path(csv, name: 'inputs/csv*/*')
+    val mode
+    val in_format
```

Comment:
Could that not be inferred by the first file in the `csv` tuple? EDIT: Ah although I guess some people (libreoffice _shakesfist_) stores TSV files with a csv suffix now I think about it...